lib/puppet/indirector/yaml.rb

```diff
@@ -19,17 +19,20 @@ def find(request)
   def save(request)
     raise ArgumentError.new("You can only save objects that respond to :name") unless request.instance.respond_to?(:name)
 
-    file = path(request.key)
+    files = path(request.key)
```

Comment:
I'm a little reluctant to make this change. It breaks the indirector model where find, etc implement CRUD operations for a single object. Also `lib/puppet/indirector/yaml.rb` is the base implementation for all yaml-based indirected models, not just reports...

Is there a reason we can't just copy the report to the other place after saving it?

Alternatively maybe a `composite` based terminus that extends terminus and contains N termini?